c/serializer/samples/remote_monitoring/remote_monitoring.c

```diff
@@ -20,16 +24,32 @@ static const char* hubSuffix = "[IoTHub Suffix, i.e. azure-devices.net]";
 // Define the Model
 BEGIN_NAMESPACE(Contoso);
 
+#ifndef WINCE
 DECLARE_STRUCT(SystemProperties,
     ascii_char_ptr, DeviceID,
     _Bool, Enabled
+	Bool, Enabled
 );
+#else
+DECLARE_STRUCT(SystemProperties,
+ascii_char_ptr, DeviceID,
+bool, Enabled
```

Comment:
_Bool can be easily defined to bool for CE in crt_abstractions.h. That way it gets used everywhere as it should be, and the #ifdef is in only one place.